README.md

```diff
@@ -25,6 +29,16 @@ For this project, we have designed it with the intention of hosting everything o
 
 ### ✏️ Design Decisions
 
+#### PDF Report Generator
+Reports are generated inside a lambda function and are stored in the S3 bucket “report storage bucket”. This function is triggered once every week, which is enough data for a valuable summary to be generated, such as average magnitude and depth.
+Users that wish to access a weekly report can do so through the dashboard, as we plan to have a page dedicated to this. This allows us to create a service that gives users the ability to access reports not just from this week, but from previous ones too.
+#### Earthquake Dashboard Service
+This dashboard service reads from the postgres Relational Database (RDS) to display key statistics regarding the data available (mean values, significant earthquakes, etc.). Because we want this service to be available at all times, we are choosing to run it as an Elastic Container Service (ECS). It also reads from an S3 bucket which contains every weekly report since the genesis of this system, so as to provide users with a downloadable PDF for every week.
+#### Earthquake User API Service
+This is run as an ECS, where the API is hosted by a Fargate instance, as the API is a constantly running service
```

Comment:
This seems a little brief compared to the other sections. Could there be more information added?

Reply:
I understand this pull request is for the dashboard README, so this should be a separate issue to address